build/x86_64_centos8/Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM registry.access.redhat.com/ubi8/ubi:latest
+FROM centos:8
 
-ADD https://www.centos.org/keys/RPM-GPG-KEY-CentOS-Official /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Official
 ADD https://rpms.remirepo.net/RPM-GPG-KEY-remi2018 /etc/pki/rpm-gpg/RPM-GPG-KEY-remi.el8
-COPY centos8.repo remi.repo /etc/yum.repos.d/
+COPY remi.repo /etc/yum.repos.d/
 
 RUN yum -y update \
+ && dnf -y install 'dnf-command(config-manager)' \
```

Comment:
This seems new. Can you give some info on what it (or PowerTools) is used for?

Reply:
PowerTools is not new — it used to be in `centos8.repo` before. Now it's bundled into the base image, but not enabled.

`dnf-command(config-manager)` adds support for the `dnf config-manager` (a.k.a. `yum config-manager`) command. I can add a comment, but IMO it would simply repeat what this line says, e.g.:
```
  # Add support for the config-manager dnf command.
  && dnf -y install 'dnf-command(config-manager)' \
```
Thoughts?

Reply:
It's helpful enough for me to understand that it was installed before and now it needs to be installed explicitly, thanks!

Reply:
Done. PTAL.